packages/askar/src/utils/askarKeyTypes.ts

```diff
@@ -7,15 +7,38 @@ export enum AskarKeyTypePurpose {
 }
 
 const keyTypeToAskarAlg = {
-  [KeyType.Ed25519]: KeyAlgs.Ed25519,
-  [KeyType.X25519]: KeyAlgs.X25519,
-  [KeyType.Bls12381g1]: KeyAlgs.Bls12381G1,
-  [KeyType.Bls12381g2]: KeyAlgs.Bls12381G2,
-  [KeyType.Bls12381g1g2]: KeyAlgs.Bls12381G1G2,
-  [KeyType.P256]: KeyAlgs.EcSecp256r1,
-  [KeyType.K256]: KeyAlgs.EcSecp256k1,
-} as const
+  [KeyType.Ed25519]: {
+    keyAlg: KeyAlgs.Ed25519,
+    purposes: [AskarKeyTypePurpose.KeyManagement, AskarKeyTypePurpose.Signing],
+  },
+  [KeyType.X25519]: {
+    keyAlg: KeyAlgs.X25519,
+    purposes: [AskarKeyTypePurpose.KeyManagement, AskarKeyTypePurpose.Signing],
+  },
+  [KeyType.Bls12381g1]: {
+    keyAlg: KeyAlgs.Bls12381G1,
+    purposes: [AskarKeyTypePurpose.KeyManagement],
+  },
+  [KeyType.Bls12381g2]: {
+    keyAlg: KeyAlgs.Bls12381G2,
+    purposes: [AskarKeyTypePurpose.KeyManagement],
+  },
+  [KeyType.Bls12381g1g2]: {
+    keyAlg: KeyAlgs.Bls12381G1,
+    purposes: [AskarKeyTypePurpose.KeyManagement],
+  },
+  [KeyType.P256]: {
+    keyAlg: KeyAlgs.EcSecp256r1,
+    purposes: [AskarKeyTypePurpose.KeyManagement],
+  },
+  [KeyType.K256]: {
+    keyAlg: KeyAlgs.EcSecp256k1,
+    purposes: [AskarKeyTypePurpose.KeyManagement],
+  },
```

Comment:
Wait I messed up I think. We should support signing and key management for the P/K 256

Reply:
Can you address it for both?

Reply:
Sure I will add them